### PR TITLE
[INT2-117] done

### PR DESCRIPTION
**Pull request by Getmegone**
First translation so better check it)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+Вся экосистема Yearn контролируется держателями токенов YFI, которые вносят предложения и голосуют "за" или "против" них, напрямую влияя на развитие Yearn. Предложения, получающие поддержку большинства \(&gt;более 50% от числа проголосовавших\), внедряются 9 подписантами кошелька с функцией мульти - подписи. Данные изменения должны быть подписаны 6 из 9 подписантов для реализации предложения. Подписанты были избраны держтелями токенов YFI в ходе голосования и могут быть смещены с этого поста в ходе дальнейших голосований. Больше информации о [процессе управления](https://docs.yearn.finance/resources/faq#governance) в целом, и о [подписантах в частности](https://docs.yearn.finance/resources/faq#who-is-on-the-multisig), вы можете найти в нашем FAQ. 
 
-## Communication Channels
+## Каналы для связи
 
 Governance Forum [https://gov.yearn.finance/](https://gov.yearn.finance/)
```
Comment (philburrrt) [inline README.md:23]: Looks good! Wondering if you missed line 23.. 'Governance Forum' looks to still be in english.

Comment (DarkGhost7) [reply README.md:23]: I gave the same feedback haha

Comment (Getmegone) [reply README.md]: Already changed and commited changes

**Comment by philburrrt** [comment]
#531

**Pull request merged**